classes/class-wc-connect-shipping-label.php

```diff
@@ -202,9 +175,18 @@ public function get_selected_packages( WC_Order $order ) {
 							$formatted = WC_Connect_Compatibility::instance()->get_formatted_variation( $product, true );
 							$product_data['attributes'] = $formatted;
 						}
+						$customs_info = get_post_meta( $product_data['product_id'], 'customs_info', true );
+						if ( $customs_info ) {
+							$product_data = array_merge( $product_data, $customs_info );
+						}
 					} else {
 						$product_data['name'] = WC_Connect_Compatibility::instance()->get_product_name_from_order( $product_data['product_id'], $order );
 					}
+					$product_data['value'] = WC_Connect_Compatibility::instance()->get_product_price_from_order( $product_data['product_id'], $order );
+					if ( ! $product_data['value'] ) {
+						$product_data['value'] = 0;
+					}
+					$product_data['parent_product_id'] = ( $product && $product->is_type( 'variation' ) ) ? $product->get_parent_id() : $product_data['product_id'];
```

Comment:
This can use `WC_Connect_Compatibility::instance()->get_parent_product_id()`. Also - where is the `parent_product_id` used?

Reply:
It's not used anymore. It was used to display a link to each product in the Customs form, but that column is no longer present in the UI. I've removed this in https://github.com/Automattic/woocommerce-services/pull/1382/commits/3920357bd3baa7236d74be395735db1f9e5635c1